helm_deploy/hmpps-accredited-programmes-api/values.yaml

```diff
@@ -33,6 +33,8 @@ generic-service:
     hmpps-accredited-programmes-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
       SENTRY_DSN: "SENTRY_DSN"
+      SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_PRISONS-API_CLIENT-ID: "SYSTEM_CLIENT_ID"
```

Comment:
I think the keys should be `SYSTEM_CLIENT_ID` and `SYSTEM_CLIENT_SECRET` to match what's in our kubernetes secrets, otherwise we'd need to add these in, right?